darts/datasets/__init__.py

```diff
@@ -648,3 +648,168 @@ def _to_multi_series(self, series: pd.DataFrame) -> List[TimeSeries]:
+
+
+class ExchangeRateDataset(DatasetLoaderCSV):
+    """
+    The collection of the daily exchange rates of eight foreign countries including Australia, British, Canada,
+    Switzerland, China, Japan, New Zealand and Singapore ranging from 1990 to 2010.
+
+    References
+    ----------
+    .. [1] https://github.com/laiguokun/multivariate-time-series-data
+    """
+
+    def __init__(self, multivariate: bool = True):
+        """
+        Parameters
+        ----------
+        multivariate: bool
+            Whether to return a single multivariate timeseries - if False returns a list of univariate TimeSeries. Default is True.
+        """
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "exchange_rate.csv",
+                uri="https://raw.githubusercontent.com/unit8co/darts/Improvement/Add_new_datasets_617/datasets/exchange_rate.csv",
+                hash="9219e9a03eb54c6e40d7eb1c9b3b6f7c",
+                header_time="Date",
+                format_time="%Y-%m-%d",
+                freq="D",
+                multivariate=multivariate,
+            )
+        )
+
+    def _to_multi_series(self, series: pd.DataFrame) -> List[TimeSeries]:
+        """
+        Load the ExchangeRateDataset dataset as a list of univariate timeseries, one for each country.
+        """
+        return [
+            series[label]
+            for label in _build_tqdm_iterator(
+                series, verbose=False, total=len(series.columns)
+            )
+        ]
+
+
+class TrafficDataset(DatasetLoaderCSV):
+    """
+    The raw data is in http://pems.dot.ca.gov. The data in this repo is a collection of 48 months (2015-2016)
+    hourly data from the California Department of Transportation. The data describes the road occupancy rates (between 0 and 1)
+    measured by different sensors on San Francisco Bay area freeways.
+
+    References
+    ----------
+    .. [1] https://github.com/laiguokun/multivariate-time-series-data
+    """
+
+    def __init__(self, multivariate: bool = True):
+        """
+        Parameters
+        ----------
+        multivariate: bool
+            Whether to return a single multivariate timeseries - if False returns a list of univariate TimeSeries. Default is True.
+        """
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "traffic.csv",
+                uri="https://raw.githubusercontent.com/unit8co/darts/Improvement/Add_new_datasets_617/datasets/traffic.csv",
+                hash="a2105f364ef70aec06c757304833f72a",
+                header_time="Date",
+                format_time="%Y-%m-%d %H:%M:%S",
+                freq="1H",
+                multivariate=multivariate,
+            )
+        )
+
+    def _to_multi_series(self, series: pd.DataFrame) -> List[TimeSeries]:
+        """
+        Load the TrafficDataset dataset as a list of univariate timeseries, one for each ID.
+        """
+        return [
+            series[label]
```

Comment:
This seems to be returning a pandas Series, we need to return a `TimeSeries`. You can probably simply do `TimeSeries.from_series(series[label])`.